app/controllers/foreman_virt_who_configure/api/v2/configs_controller.rb

```diff
@@ -52,7 +52,6 @@ def deploy_script
             param :hypervisor_password, String, :desc => N_("Hypervisor password, required for all hypervisor types except for libvirt"), :required => false
             param :satellite_url, String, :desc => N_("Satellite server FQDN"), :required => true
             param :debug, :bool, :desc => N_("Enable debugging output")
-            param :proxy, String, :desc => N_('HTTP proxy that should be used for communication between the server on which virt-who is running and the hypervisors and virtualization managers.'), :required => false
```

Comment:
I think it needs to accept an `http_proxy_id` now. Likely needs a code another change in the controller to actually make it work. Make sure to add a controller test for it (maybe an existing test can be modified!)

Reply:
@jturel++

@chris1984, you can look at https://github.com/theforeman/foreman_virt_who_configure/blob/a2faa076e064f17b30f3d49a0e1aa5603f26f45f/app/models/foreman_virt_who_configure/config.rb#L6 to be sure.

Also without a param listed here hammer will not create an option for users to be able create a config with proxy.

Reply:
Updated to http_proxy_id, integer